### PR TITLE
ICDC-1741

### PR DESCRIPTION
**Pull request by tyudosen**
Added tooltip for the theme switch button on the cases page

### CODE REVIEW

**Review by rana22 — APPROVED** [review]
LGTM

File: src/bento/dashboardData.js [hunk 1]
```diff
@@ -1,6 +1,13 @@
 /* eslint-disable */
 import gql from 'graphql-tag';
 
+// --------------- Dashboard Data --------------------
+export const dashboardData = {
```
Comment (iksheth) [inline src/bento/dashboardData.js:5]: can you rename "dashboardData" to themeToggleTooltip ?

**Review by iksheth — CHANGES_REQUESTED** [review]

File: src/bento/dashboardData.js (hunk 1 above)
Comment (tyudosen) [reply src/bento/dashboardData.js]: done

**Review by iksheth — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**